docs/Spark/gems/transform/dynamic-select.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 13
+title: Dynamic Select
+id: dynamic-select
+description: Dynamically filter columns of your dataset based on a set of conditions.
+tags:
+  - gems
+  - filter
+  - dynamic
+---
+
+Use the Dynamic Select Gem to dynamically filter columns of your Dataset based on a set of conditions.
+
+## Configuration
+
+There are two ways to configure the Dynamic Select.
+
+| Configuration         | Description                                                                                   |
+| --------------------- | --------------------------------------------------------------------------------------------- |
+| Select field types    | Choose one or more types of columns to keep in the Dataset, such as string, decimal, or date. |
+| Select via expression | Create an expression that limits the type of columns to keep in the Dataset.                  |
+
+## Examples
+
+You’ll use Dynamic Select when you want to avoid hard-coding your choice of columns. In other words, rather than define each column to keep in your Pipeline, you let the system automatically choose the columns based on certain conditions or rules.
+
+### Remove date columns using field type
+
+Assume you would like to remove irrelevant date and timestamp columns from your Dataset. You can do so with the **Select field types** method by selecting all field types to maintain, except for date and timestamp.
+
+![Keep all columns except Date and Timestamp column using the visual interface](./img/remove-date-timestamp.png)
+
+### Remove date columns with an expression
+
+Using the same example, you can accomplish the same task with the **Select via expression** method by inputting the the expression `column_type NOT IN ('date', 'timestamp')`.
```

Comment:
Is the Select via expression the same kind of method used in the Bulk Column Rename example?

Reply:
Good question! I asked the same to Ashish. They are not the same. For both of them, you only have access to a limited number of variables (which actually are included in the UI for dynamic select). Ashish asked me to open an Asana ticket to ask him to include the available variables for bulk column rename.
<img width="650" alt="image" src="https://github.com/user-attachments/assets/ea6f4d26-c4d1-472d-82d2-d32a21b38188">